google-cloud-spanner/src/test/java/com/google/cloud/spanner/DatabaseClientImplTest.java

```diff
@@ -1548,4 +1550,40 @@ public void testReadDoesNotIncludeStatement() {
       assertThat(e.getMessage()).doesNotContain("Statement:");
     }
   }
+
+  @Test
+  public void testSpecificTimeout() {
+    mockSpanner.setExecuteStreamingSqlExecutionTime(
+        SimulatedExecutionTime.ofMinimumAndRandomTime(10000, 0));
+    final DatabaseClient client =
+        spanner.getDatabaseClient(DatabaseId.of(TEST_PROJECT, TEST_INSTANCE, TEST_DATABASE));
+    Context.current()
+        .withValue(
+            SpannerOptions.CALL_CONTEXT_CONFIGURATOR_KEY,
+            SpannerCallContextTimeoutConfigurator.create()
+                .withExecuteQueryTimeout(Duration.ofNanos(1L)))
```

Comment:
So this given timeout is for every ExecuteQuery inside this context? If two ExecuteQuery calls are inside this context, each of them will be limited by the 1ns timeout, right?

Reply:
Correct.